programs/KJVDataDump/main.cpp

```diff
@@ -371,14 +396,44 @@ int main(int argc, char *argv[])
 						if (bOutputTemplates) {
 							std::cout << pVerse->m_strTemplate.toUtf8().data() << std::endl;
 						}
-						if (bOutputVerseText) {
-							if (bOutputTemplates) std::cout << strSpacer.toUtf8().data();
-							if (bOutputTransChangeAdded) {
-								std::cout << lstVerseWords.join(QChar(' ')).toUtf8().data();
-							} else {
-								std::cout << strParsedVerse.toUtf8().data();
+						if (bOutputVerseText && nLetterCount > 0) {
```

Comment:
This, of course, essentially breaks the functionality of this tool to be a tool that dumps out Bible database content, if it can't output verse text unless you've specifically given it a letter count.

For this to be part of this tool long-term, there would need to be two paths here -- one that just outputs verse text as it always did and one that outputs text for this special letter search mode.